GlobalQuakeServer/src/main/java/gqserver/websocketserver/JettyServer.java

```diff
@@ -0,0 +1,104 @@
+package gqserver.websocketserver;
+
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.HttpConnectionFactory;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.server.handler.HandlerList;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.websocket.server.config.JettyWebSocketServletContainerInitializer;
+
+import org.tinylog.Logger;
+
+import globalquake.core.Settings;
+import gqserver.websocketserver.handler_chain.DropConnectionHandler;
+import gqserver.websocketserver.handler_chain.ServerHeader;
+import gqserver.websocketserver.handler_chain.HttpCatchAllLogger;
+
+
+public class JettyServer {
+    private static Server server;
+    private static ServerConnector connector;
+
+
+    //ALWAYS HAVE THIS LINE LAST SO OTHER STATIC VARIABLES ARE INITIALIZED

```

Comment:
by having the JettyServer object as a field in WebSocketEventServer you can avoid hassle like this ;)